about.md

```diff
@@ -15,18 +15,32 @@ Please join us on our:
 for day-to-day advice, help, brainstorming, news, networking, and socialising.
 <!-- TODO: add a mailing list. - [**mailing list**](https://) for announcements, calls, and notices. -->
 <!-- - [**instagram**]() -->
-- [**X/Twitter**](http://twitter.com/haam_community), [**mastodon**](https://genomic.social/@haam_community) or [**bluesky**]().
+- [**Twitter**](http://twitter.com/haam_community), [**mastodon**](https://genomic.social/@haam_community) or [**bluesky**](https://bsky.app/profile/haam-community.bsky.social).
 
 to get involved and keep up-to-date or contact us through [email](mailto:haam.community2023@gmail.com)! ⭐️
 
 ____________________________________________________________________
 
-If we are not the community you are looking for, check out other [ISBA-related communities](https://www.isbarch.org/affiliations-and-initiatives) working with ancient biomolecules or related research questions:
+<div id="line23">
+  <h2>Other communities:</h2>
+  <p>If we are not the community you are looking for, check out other 
```

Comment:
Id rephrase that to something like:
`You might also be interested in these other bioarchaeological communities`